tests/spec/core/link-to-dfn-spec.js

```diff
@@ -25,9 +25,12 @@ describe("Core — Link to definitions", () => {
       <section data-link-for="Request">
         <h2><dfn>Request</dfn> interface</h2>
         <pre class="idl">
-          interface Request {};
+          interface Request {
+            Request clone();
+          };
         </pre>
-        <p id="codeWrap">A <a>Request</a> object.</p>
+        <p id="codeWrap">A <a>Request</a> object has a <dfn>clone</dfn> method.</p>
+        <p id="codeWrapMethod"><a>clone</a>, <a>clone()</a>, <a>Request.clone</a>, and <a>Request.clone()</a> are all same.</p>
         <p id="noCodeWrap">An instance of <a lt="Request">the request interface</a>.</p>
```

Comment:
Related... looks like this test is wrong. 
```suggestion
        <p id="noCodeWrap">An instance of <a data-lt="Request">the request interface</a>.</p>
```